x-pack/plugins/cases/public/components/create/form_context.test.tsx

```diff
@@ -612,11 +620,32 @@ describe('Create case', () => {
         onChoicesSuccess(useGetChoicesResponse.choices);
       });
 
+      await waitFor(() => {
+        expect(
+          within(screen.getByTestId('severitySelect')).getByRole('option', { name: '2 - High' })
```

Comment:
Can't we move all these `screen.getByTestId(...)` to variables?

```
const severitySelect = screen.getByTestId('severitySelect');
const urgencySelect = screen.getByTestId('urgencySelect');
const impactSelect = screen.getByTestId('impactSelect');

await waitFor(() => {
  expect(within(severitySelect).getByRole('option', { name: '2 - High' }));

  expect(within(urgencySelect).getByRole('option', { name: '2 - High' }));

  expect(within(impactSelect).getByRole('option', { name: '2 - High' }));

  expect(screen.getByRole('option', { name: 'Software' }));
});

[severitySelect, urgencySelect, impactSelect].forEach((subj) => {
  userEvent.selectOptions(subj, ['2']);
});
```

Reply:
Fixed!